setup.py

```diff
@@ -1,10 +1,43 @@
 import setuptools
 import os
 
-
 def read(fname):
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
+base_packages = [
+    "scikit-learn>=0.22.2",
+    "pandas>=1.0.0",
+    "matplotlib>=3.1.1",
+    "scipy>=1.4.0",
+    "joblib>=0.13.2",
+    "tqdm>=4.41.0",
+    "shap>=0.38.1",
+    "numpy>=1.19.0"
+]
+
+dev_dep = [
+    "flake8>=3.8.3",
+    "black>=19.10b0",
+    "pre-commit>=2.5.0",
+    "mypy>=0.770",
+    "flake8-docstrings>=1.4.0",
```

Comment:
Once we have these here we can remove tests/requirements_test.txt, and in github workflows, install probatus["testing"] or something similar.
What do you think?